gst/tensor_transform/tensor_transform.c

```diff
@@ -735,6 +758,58 @@ gst_tensor_transform_transpose (GstTensor_Transform * filter,
 }
 
 
+/**
+ * @brief subrouting for tensor-tranform, "stand" case.
+ *        : pixel = abs((pixel - average(tensor))/(std(tensor) + val))
+ * @param[in/out] filter "this" pointer
+ * @param[in] inptr input tensor
+ * @param[out] outptr output tensor
+ * @return Gst flow status
+ */
+static GstFlowReturn
+gst_tensor_transform_stand (GstTensor_Transform * filter,
+    const uint8_t * inptr, uint8_t * outptr)
+{
+  int i;
+  size_t Size;
+  uint32_t *fromDim = filter->fromDim;
+  float average, stand;
```

Comment:
For float(32) elements, why don't you use double(64) average/std to make it more accurate and less prone to overflow?